cfme/tests/networks/test_sdn_inventory_collection.py

```diff
@@ -40,10 +40,6 @@ def test_sdn_api_inventory_networks(provider, appliance):
 
 
 @pytest.mark.provider([AzureProvider, EC2Provider], override=True, scope='function')
-@pytest.mark.meta(blockers=[BZ(1550605, forced_streams=["5.9", "5.8"],
```

Comment:
Might be useful to keep the BZs in the docsting, so it will be easier to determine regression

Reply:
Thanks, missed this one, tried to add them to the docblocks as I removed BZ's.